### DISCUSSION TITLE
Multiple tenant hosts in local development

### DISCUSSION THREAD

**Discussion by mariojsnunes** [OrchardCMS/OrchardCore#7266] [Q&A]
Hello guys,

Is there any guide for setting up the local development with multiple tenants that vary by hostname? (not querystring)
I'm working on windows, so I guess I would need to alter the hosts file and point multiple localhost ports to the main app?

Thanks

**Comment by jptissot** [comment]
I don't think you need multiple ports. simply multiple entries in the hosts file.

**Reply by mariojsnunes** [reply]
I still haven't figured out how to get it working.
So when I run my project with visual studio 2019, it starts up at https://localhost:44313

I've added this line to hosts file: _localhost:44313 test.dev_

I read somewhere that the hosts file doesn't accept ports which explains why it's not working

**Reply by deanmarcussen** [reply]
a hosts file would look like this
##
127.0.0.1	test.me

I would recommend not using .dev as chrome took that over a while ago and caused all sort of pain for devs everywhere.

You can't use ports on a dns entry, they're a different thing.

https://readme.localtest.me/ is quite useful to save having to add host entries.

You'll also have to bind iisexpress to the dns entry.

Here is a related stackoverflow
https://stackoverflow.com/questions/4709014/using-custom-domains-with-iis-express
